bokehjs/src/lib/models/glyphs/image.ts

```diff
@@ -73,10 +73,15 @@ export class ImageView extends XYGlyphView {
   }
 
   _lrtb(i: number) : [number, number, number, number]{
-    const l = this._x[i]
-    const r = l + this._dw[i]
-    const b = this._y[i]
-    const t = b + this._dh[i]
+    const xr = this.renderer.xscale.source_range
+    const x1 = this._x[i]
+    const x2 = xr.is_reversed ? x1 - this._dw[i] : x1 + this._dw[i]
+
+    const yr = this.renderer.yscale.source_range
+    const y1 = this._y[i]
+    const y2 = yr.is_reversed ? y1 - this._dh[i] : y1 + this._dh[i]
```

Comment:
style nit: would say either add a blank line after these three "y" lines of code, or don't add one after the three "x" lines of code

Reply:
Addressed in 9fd3fd32d